ThunderCloud/ContentController.swift

```diff
@@ -1256,10 +1264,12 @@ public extension ContentController {
         var thinnedAssetName = URL(fileURLWithPath: file).lastPathComponent
         let lastUnderScoreComponent = thinnedAssetName.components(separatedBy: "_").last
         
+        let extensions = [".jpg", ".jpeg", ".png"]
```

Comment:
Not sure if we want to `lowercase()` for safety, do you think that matters with paths?